### PR TITLE
pkp/pkp-lib#950 Remove multi-context submission button

### PR DESCRIPTION
**Pull request by NateWr**

### CODE REVIEW

File: templates/dashboard/submissions.tpl [hunk 1]
```diff
@@ -6,40 +6,18 @@
...
-				</li>
-			</ul>
-		</form>
+		<ul>
+			<li>
+				<a href="{url context=$context->getPath() page="submission" op="wizard" escape=false}" class="new_submission">
+					{translate key="submission.submit.newSubmissionSingle"}
+				</a>
+			</li>
+		</ul>
```
Comment (beghelli) [inline templates/dashboard/submissions.tpl]: i think you will still need the context check, since editors only cannot submit, I think.

Comment (NateWr) [inline templates/dashboard/submissions.tpl]: Is it the `$accessibleContexts` check [here](https://github.com/pkp/pkp-lib/pull/964/files#diff-3ff98b5293cf8bfac6aab3f1d45be01cL81) that needs to remain? Is there an easier, single-context way to check if a user can submit?

Comment (beghelli) [inline templates/dashboard/submissions.tpl]: The user needs to be anything other than just editor, I think. I am out getting Helena, but will be back in a few minutes, I can take a better look.

-----Mensagem Original-----
De: "Nate Wright" <notifications@github.com>
Enviada em: ‎14/‎12/‎2015 15:03
Para: "pkp/pkp-lib" <pkp-lib@noreply.github.com>
Cc: "Bruno Beghelli" <bruno.beghelli@gmail.com>
Assunto: Re: [pkp-lib] pkp/pkp-lib#950 Remove multi-context submission button(#964)

In templates/dashboard/submissions.tpl:
> -						{capture assign="defaultLabel"}{translate key="context.select"}{/capture}
> -						{fbvElement type="select" id="multipleContext" from=$contexts defaultValue=0 defaultLabel=$defaultLabel translate=false size=$fbvStyles.size.MEDIUM}
> -					{elseif $contextCount == 1}
> -						{capture assign="singleLabel"}{translate key="submission.submit.newSubmissionSingle"}{/capture}
> -						{fbvElement type="button" id="singleContext" label=$singleLabel translate=false}
> -					{/if}
> -				</li>
> -			</ul>
> -		</form>
> +		<ul>
> +			<li>
> +				<a href="{url context=$context->getPath() page="submission" op="wizard" escape=false}" class="new_submission">
> +					{translate key="submission.submit.newSubmissionSingle"}
> +				</a>
> +			</li>
> +		</ul>
Is it the $accessibleContexts check here that needs to remain? Is there an easier, single-context way to check if a user can submit?

Comment (beghelli) [inline templates/dashboard/submissions.tpl]: @NateWr, I think that everyone that has access to the dashboard should be able to submit. So you just need to get the current context, as you did already. Sorry for delaying this PR. :)

**Pull request closed without merging**